cloud

```diff
@@ -1264,6 +1266,7 @@ function cloud_functions_env() {
     eval "$params"
     declare -A env=(
         [KCIDB_LOG_LEVEL]="$log_level"
+        [KCIDB_OPTIMIZE]="$optimize"
```

Comment:
You're setting up to create an environment variable called `KCIDB_OPTIMIZE` here. However, we need to create an environment variables called `PYTHONOPTIMIZE`. The Python interpreter looks for it in the environment to decide on how much optimization it should do. See the documentation here: https://docs.python.org/3/using/cmdline.html#envvar-PYTHONOPTIMIZE